### PR TITLE
Mousewheel sensitivity more appropriate for mac

### PR DESCRIPTION
**Pull request by celiasmith**
(and unchanged for other oss)  The results of discussion in #226

### CODE REVIEW

**Comment by jgosmann** [comment]
Nope. :no_good: 
I slightly move my fingers and the trackpad and the network graph disappears into nirvana.

**Comment by jgosmann** [comment]
Same for the mouse wheel and both Chrome and Firefox.
This is like the worst version yet. :fire:

File: nengo_viz/static/viz_viewport.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ VIZ.Viewport = function() {
     this.x = 0;
     this.y = 0;
     this.scale = 1.0;
-    
-    this.scale_step_size = 1.1;
-    
+
+    this.scale.step_size = 1.1;
```
Comment (jgosmann) [inline nengo_viz/static/viz_viewport.js]: This should probably be `this.scale_step_size`.

File: nengo_viz/static/viz_viewport.js [hunk 2]
```diff
@@ -2,9 +2,11 @@ VIZ.Viewport = function() {
...
+    if (navigator.appVersion.indexOf("Mac")!=-1) {
+        this.scale.step_size = 1.05;
```
Comment (jgosmann) [inline nengo_viz/static/viz_viewport.js]: Same here.

**Comment by celiasmith** [comment]
You'll notice it was actually broken before i changed anything... I have no idea what's going on.  If I comment those three changes out, it's still broken...?  But, it seems fine in master!  I'll probably have to try again, as this is baffling.

File: nengo_viz/static/viz_viewport.js (hunk 2 above)
Comment (celiasmith) [inline nengo_viz/static/viz_viewport.js]: yes

**Comment by tcstewar** [comment]
This has been replaced by #247 which has now been merged.

**Pull request closed without merging**